packages/web/src/scss/components/Dropdown/index.html

```diff
@@ -0,0 +1,7 @@
+{{#> layout/plain }}
+
+<div class="Dropdown is-open">
+  Content
```

Comment:
I would prefer to add more content. Now it looks like some kind of weird button in the demo page.

Reply:
Link items added.